src/components/lib/styles/__style-guide.css

```
@@ -140,6 +140,17 @@ h5 {
   line-height: 1.125em;
 }
 
+a {
+  font-family: var(--font-body);
+  font-size: 1rem;
+  font-weight: 700;
+  line-height: 1.5;
+  color: var(--primary);
+  text-decoration: underline;
+  text-underline-offset: 2px;
+  cursor: pointer;
```

We can skip properties that are default for an `a` tag.

```suggestion
  font-family: var(--font-body);
  font-weight: 700;
  line-height: 1.5;
  color: var(--primary);
  text-underline-offset: 2px;
```